frontend/src/hooks/api/secretImports/queries.tsx

```diff
@@ -160,10 +160,9 @@ export const useGetImportedSecretsAllEnvs = ({
         projectId,
         path
       }),
-      queryFn: () => fetchImportedSecrets(projectId, env, path).catch(() => []),
-      enabled: Boolean(projectId) && Boolean(env),
-      // eslint-disable-next-line react-hooks/rules-of-hooks
-      select: (data: TImportedSecrets[]) => {
```

Comment:
`select` runs every render so with thousands of secrets this clogs up the thread - move to `queryFn` to perform once.